LondonBritishLibrary/orient/BLorient674.xml

```diff
@@ -109,16 +109,19 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                             
                                    <handDesc>
                             <handNote script="Ethiopic" xml:id="h1">
-                                <desc>Written in a good hand. </desc>
+                                <desc>Written in a good hand.</desc>
                                 <date notBefore="1650" notAfter="1699"/>
                             </handNote>
                         </handDesc>   
                             
                             <additions>
                                 <list>
                                     <item xml:id="a1">
-                                        <locus target="#"/>
-                                        <desc type=""></desc>
+                                        <locus target="#1r"/>
+                                        <desc type="Inventory">Several Amharic inventories for a church of 
+                                            ʾArbāʿtu ʾƎnsǝsā, from the time of ʾaṣe Takla Hāymānot,
```

Comment:
this could be repeated under provenance, as the MS most probably used to belong to the church of ʾArbāʿtu ʾƎnsǝsā in Gondar (LOC1526Arbaet)